infrastructure/dev.tfvars

```
@@ -0,0 +1,9 @@
+environment         = "dev"
+location            = "uksouth"
+resource_group_name = "aiacademy25-dev"
```

```suggestion
resource_group_name = "aiacademy25-dev"
```

would maybe name this something more relevant to your team / application